lib/is-database-ready.js

```diff
@@ -0,0 +1,37 @@
+const Postgrator = require('postgrator')
+const getConfig = require('./config')
+
+function getPostgratorInstance(config) {
+  return new Postgrator({
+    driver: 'pg',
+    host: config.pgPlugin.write,
+    port: config.pgPlugin.config.port,
+    database: config.pgPlugin.config.database,
+    username: config.pgPlugin.config.user,
+    password: config.pgPlugin.config.password,
+    ssl: config.pgPlugin.config.ssl,
+    schemaTable: 'migrations',
+    currentSchema: 'public'
+  })
+}
+
+async function isDatabaseReady() {
+  try {
+    const config = await getConfig()
+    const postgrator = getPostgratorInstance(config)
+    const version = await postgrator.getDatabaseVersion()
+
+    console.log('Database is Ready: Version', version)
+    process.exit(0)
+  } catch (error) {
+    console.error(error)
+    console.log('Database is not Ready')
+    process.exit(1)
```

Comment:
Don't call `process.exit()` from within an async function. Instead, set the `process.exitCode` here if absolutely necessary. Or, likely better, have this just return `false` since the semantic is: If there's an error the database is not ready.